ln/subscribe_invoice.ts

```diff
@@ -47,18 +51,20 @@ const subscribeInvoice = async (bot, id, resub) => {
             rate
           );
         }
-        order.invoice_held_at = Date.now();
+        order.invoice_held_at = new Date();
         order.save();
       }
       if (invoice.is_confirmed) {
         const order = await Order.findOne({ hash: id });
+        if (order === null) return;
```

Comment:
@Mersho why are you changing behaviour here? converting from JS to TS should not change the behaviour of the program